app/mailers/notification_mailer.rb

```diff
@@ -1,13 +1,15 @@
-# frozen_string_literal: true
-
-# Send notification email based on the email address and message from the db
+require_relative 'application_mailer'
```

Comment:
You don't need to require this one, Rails automagically takes care of autoloading constants 🎩